internal/cmd/ini/common.go

```diff
@@ -105,16 +106,21 @@ func completeBasic(toComplete string) []string {
 	return files
 }
 
-func extValidator(framework string) survey.Validator {
+func extValidator(framework, frameworkVersion string) survey.Validator {
 	var exts []string
 	switch framework {
 	case espresso.Kind:
 		exts = []string{".apk", ".aab"}
 	case xcuitest.Kind:
 		exts = []string{".ipa", ".app"}
 	case cypress.Kind:
-		exts = []string{".json"}
+		if getMajorVersion(frameworkVersion) < 10 {
+			exts = []string{".json"}
+			break
+		}
+		exts = []string{".js", ".ts", ".mjs", ".cjs"}
```

Comment:
A bit simpler without the need for `break`s:

```suggestion
		exts = []string{".js", ".ts", ".mjs", ".cjs"}
		if getMajorVersion(frameworkVersion) < 10 {
			exts = []string{".json"}
		}
```